src/serve_rendered.js

```diff
@@ -481,8 +481,12 @@ module.exports = function(options, repo, params, id, publicUrl, dataResolver) {
           if (!buffer) {
             return res.status(404).send('Not found');
           }
-
+          var hdr_center = 'lat=$lat,lon=$lon'
+            .replace('$lat', lat)
+            .replace('$lon', lon);
           res.set({
+            'tileserver-Center': hdr_center,
+            'tileserver-Zoom': z,
```

Comment:
I'm used to seeing headers title cased like `Content-Type` and `X-Forwarded-For`, not `content-Type` or `x-Forwarded-For`. For consistency, maybe use `TileServer-Center` and `TileServer-Zoom`.

RFC now recommends dropping the `X-` prefix, but for application specific headers that will never graduate to web standards, the `X-` is a good indicator it's custom. https://stackoverflow.com/questions/3561381/custom-http-headers-naming-conventions